### ISSUE TITLE
Unable to configure the class documentation template

### ISSUE THREAD

**Issue by msbrm** [NilsJPWerner/autoDocstring#285]
I would like to be able to configure the class documentation template. How should I do that?
![微信截图_20240528155047](https://github.com/NilsJPWerner/autoDocstring/assets/112378987/0f346533-88b0-48ed-8f78-dd49bcd78d48)
I also hope it can support file header comments.

**Comment by mdragoss** [comment]
Yeah run into the same issues, I don't need to appear `Args` in doc strings.

**Comment by bavalpey** [comment]
Would be helpful if it by default added :ivar: for instance variables (specifically those not prefixed with underscore).
